scenes/MapScene.js

```diff
@@ -1,18 +1,24 @@
 import React, { Component } from 'react';
-import { View, Text } from 'react-native';
+import { ScrollView, Text, Image, Dimensions } from 'react-native';
 
 export default class MapScene extends Component {
```

Comment:
I think we can reduce this to just a function:
https://facebook.github.io/react/docs/components-and-props.html#functional-and-class-components